DataProcessing/datax-host/src/main/scala/datax/sink/OutputManager.scala

```diff
@@ -85,7 +85,7 @@ object OutputManager {
           val spark = df.sparkSession
           spark.synchronized{
             if(shouldGeneratorProcessedSchema){
-              HadoopClient.writeHdfsFile(processedSchemaPath, new StructType(outputColumns).prettyJson, true)
+              HadoopClient.writeHdfsFile(processedSchemaPath, new StructType(outputColumns).prettyJson, true, false)
```

Comment:
Let's use named parameter when passing the values here so it will be readable for viewers. so this should be like
```scala
HadoopClient.writeHdfsFile(processedSchemaPath, 
                                          new StructType(outputColumns).prettyJson, 
                                          overwriteIfExists = true, 
                                          directWrite = false)
```